### PR TITLE
Add .meta files

### PR DESCRIPTION
**Pull request by Sirikon**

### CODE REVIEW

**Comment by shiftkey** [comment]
@Sirikon any links to documentation about what these files represent - and why they can be ignored in source control?

File: Unity.gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 # Unity3D generated meta files
 *.pidb.meta
+*.meta
```
Comment (shiftkey) [inline Unity.gitignore]: Does this make the previous rule redundant now?

**Comment by Sirikon** [comment]
Sorry, forget this, misunderstood this thread:

http://forum.unity3d.com/threads/4-3-assets-folder-is-full-of-meta-files.211923/

**Comment by shiftkey** [comment]
@Sirikon all good, thanks for following up so quickly!

**Pull request closed without merging**